### ISSUE TITLE
Importing dependencies

### ISSUE THREAD

**Issue by svm1** [watson-developer-cloud/text-to-speech-swift#9]
Hi there,

I'm relatively new to iOS dev so please excuse me if this a basic question - just a small doubt on how the necessary TextToSpeech dependencies should be imported. 

I ran the mentioned Carthage command and had the swift-sdk install in my project directory, but XCode still doesn't seem to recognize `TextToSpeechV1`. Is there anything else I should be doing/any steps I'm missing?

**Comment by AnthonyOliveri** [comment]
Did you also copy the `Starscream` framework to your project? Under the [Dependency Management](https://github.com/watson-developer-cloud/swift-sdk#dependency-management) section of the swift-sdk README, we explain how `TextToSpeechV1` requires `Starscream`.

**Comment by svm1** [comment]
~~Yes, I already had `Starscream` imported from CocoaPods and working.~~ 
I made sure to add `Starscream` in the way mentioned just in case, still facing the same issue though. I am only receiving errors about `TextToSpeechV1` not being recognized.

The project compiles fine but outputs the following runtime error:
`dyld: Library not loaded: @rpath/TextToSpeechV1.framework/TextToSpeechV1`

I made sure to add the framework in the Build Settings, and it is listed there as a project framework, but am not sure why this is still happening. I would greatly appreciate any guidance.

**Comment by AnthonyOliveri** [comment]
Can you verify that you completed all of the steps in the [Carthage installation](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos) instructions?

**Comment by svm1** [comment]
Thanks for the prompt responses.

I indeed have completed those, I ran the update command multiple times to make sure. 
I followed the directions of [this link](https://stackoverflow.com/questions/24333981/ios-app-with-framework-crashed-on-device-dyld-library-not-loaded-xcode-6-beta), which instructed to add the frameworks as Embedded Binaries. Doing this solved the problem and let the app run without the `Library not loaded` message. 

Now I'm facing problems with fatal errors thrown by the `AVAudioPlayer`, but I suppose those are unrelated and another thing to debug.
